server/src/services/getBiodataSearch.service.ts

```
@@ -0,0 +1,127 @@
+import { CustomReq } from '../interfaces/express';
+import { PersonalAllDetailsModel } from '../models/PersonalAllDetails.Model';
+
+// Helper function to add range conditions to a field
+const addRangeToQuery = (field: string, min?: string, max?: string) => {
+  const conditions: any = {};
+  
+  if (min) {
+    conditions['$gte'] = parseInt(min);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)